### PR TITLE
React DevTools v4 blog post

### PR DESCRIPTION
**Pull request by bvaughn**
First pass at v4 release blog post. I decided to keep this pretty high level (with the details being in the [changelog](https://github.com/bvaughn/react-devtools-experimental/blob/master/CHANGELOG.md)) so that it could hopefully answer all of the most important questions at a glance.

Release date and React Native version is arbitrary. I just needed something to fill in.

[View the formatted blog post](https://deploy-preview-2199--reactjs.netlify.com/blog/)

### CODE REVIEW

**Comment by rachelnabors** [comment]
For folks who haven't been following along, what do you mean by "A lot has changed!" (since...?)

**Comment by rachelnabors** [comment]
"the extension should update " can you link to the extension for folks who haven't installed it yet?

**Comment by bvaughn** [comment]
The header above says that we're announcing a new release, so I think "a lot has changed [in the new release]" is implied, but I could add "in version 4" if you think that's better?

File: content/blog/2019-08-01-new-react-devtools.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
+title: "Introducing the new React DevTools"
```
Comment (gaearon) [inline content/blog/2019-08-01-new-react-devtools.md]: Nit: capitalize "New"? We don't always do that but mostly do for announcement posts

File: content/blog/2019-08-01-new-react-devtools.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+author: [bvaughn]
+---
+We are excited to announce a new release of the React developer tools, available today in supported browsers!
```
Comment (gaearon) [inline content/blog/2019-08-01-new-react-devtools.md]: Nit: capitalize Developer Tools?

Comment (gaearon) [reply content/blog/2019-08-01-new-react-devtools.md]: Re: "supported browsers", can we be more specific? It doesn't tell you much since obviously it's unavailable in unsupported browsers — and if you don't know e.g. that FF is supported, this doesn't tell you either. Or we could remove the qualifier altogether if you don't want to go into detail.

File: content/blog/2019-08-01-new-react-devtools.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+## What's changed?
+
+A lot has changed!
+[Check out the changelog](https://github.com/bvaughn/react-devtools-experimental/blob/master/CHANGELOG.md) for a complete list.
```
Comment (gaearon) [inline content/blog/2019-08-01-new-react-devtools.md]: People generally expect changelogs to be boring and might not look. But yours is actually pretty beautiful, even with gifs. I'd maybe copy and paste these gifs here, people like to see pretty pictures. Maybe not all of them if you would rather keep it short.

Alternatively you could make a bullet point summary for major changes that make it feel worth it ("a lot has changed" is actually a bit scary), and then conclude "To see the demos and learn about the changes in more detail, see the changelog".

File: content/blog/2019-08-01-new-react-devtools.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+At a high level, this new version should offer significant performance gains and an improved navigation experience.
+
+## Which versions of React are supported?
+
+**`react-dom`**
+
+* `0`-`14.x`: Not supported
+* `15.x`: Partial (component filters not supported)
```
Comment (gaearon) [inline content/blog/2019-08-01-new-react-devtools.md]: This looks like a downgrade. But component filters are a new feature. Maybe there's another way to phrase it to make it look additive rather than diminishing.

File: content/blog/2019-08-01-new-react-devtools.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+## How do I get the new DevTools?
+
+For those of you using DevTools in Chrome or Firefox, the extension should update automatically within the next couple of hours.
+
+Standalone users (e.g. React Native or Safari) can install the new version from NPM:
+
+```shell
+npm install -g react-devtools@^4
+```
```
Comment (gaearon) [inline content/blog/2019-08-01-new-react-devtools.md]: Maybe link to the standalone instructions here.

File: content/blog/2019-08-01-new-react-devtools.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+## Where did all of the DOM elements go?
```
Comment (gaearon) [inline content/blog/2019-08-01-new-react-devtools.md]: Thanks for this

File: content/blog/2019-08-01-new-react-devtools.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+
+## How do I get the old version back?
+
+If you are working with React Native version 60 (or older) you can install the previous release of DevTools from NPM:
+
+```shell
+npm install --dev react-devtools@^3
+```
+
+For older versions of React DOM (v14 or earlier) you will need to build the extension from source:
```
Comment (gaearon) [inline content/blog/2019-08-01-new-react-devtools.md]: v0.14 to be clearer

File: content/blog/2019-08-01-new-react-devtools.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+git clone https://github.com/facebook/react-devtools
+
+cd react-devtools
+
+# Install dependencies and build the unpacked extension
+yarn install
+yarn build:extension
+
+# Follow the on-screen instructions for installation
+```
```
Comment (gaearon) [inline content/blog/2019-08-01-new-react-devtools.md]: This leaves no opportunity to voice feedback. Maybe add "Thanks" where you say you're thankful to people who tried it early, and note that we're happy to hear feedback on twitter or in the repository. (This doesn't mean we'll act on every single piece of feedback, but we need a space for people to complain and report issues to).

File: content/blog/2019-08-01-new-react-devtools.md (hunk 1 above)
Comment (rachelnabors) [reply content/blog/2019-08-01-new-react-devtools.md]: That's what threw me off. I couldn't tell if these were new or a new version (hence my comment about "a lot has changed since..." above!)

File: content/blog/2019-08-01-new-react-devtools.md (hunk 3 above)
Comment (rachelnabors) [reply content/blog/2019-08-01-new-react-devtools.md]: I like the last one, bullets with a link.... _and_ a gif?

File: content/blog/2019-08-01-new-react-devtools.md (hunk 2 above)
Comment (bvaughn) [reply content/blog/2019-08-01-new-react-devtools.md]: Sure. I had listed Chrome, Firefox, and Chromium Edge at first but then thought maybe it was too wordy and deleted it. I can re-add.

File: content/blog/2019-08-01-new-react-devtools.md (hunk 4 above)
Comment (bvaughn) [reply content/blog/2019-08-01-new-react-devtools.md]: Good point.

File: content/blog/2019-08-01-new-react-devtools.md (hunk 8 above)
Comment (bvaughn) [reply content/blog/2019-08-01-new-react-devtools.md]: Hm, fair. I guess I assumed people would not hesitate to tweet at us or file GH issues as they usually do. I'll add an explicit thanks section though with feedback instructions.

File: content/blog/2019-08-15-new-react-devtools.md [hunk 9]
```diff
@@ -0,0 +1,73 @@
+---
+title: "Introducing the New React DevTools"
+author: [bvaughn]
+---
+We are excited to announce a new release of the React Developer Tools, available today in Chrome, Firefox, and (Chromium) Edge!
+
+## What's changed?
+
+A lot has changed in version 4!
+At a high level, this new version should offer significant performance gains and an improved navigation experience.
```
Comment (gaearon) [inline content/blog/2019-08-15-new-react-devtools.md]: I still feel like there should be at least one flagship feature here. Maybe add "This release also offers full support for React Hooks, including inspecting nested objects."

File: content/blog/2019-08-15-new-react-devtools.md [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+**`react-native`**
+* `0`-`0.61`: Not supported
+* `0.62`: Supported
+
+## How do I get the new DevTools?
+
+React DevTools is available as an extension for [Chrome](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi?hl=en) and [Firefox](https://addons.mozilla.org/en-US/firefox/addon/react-devtools/).
+If you have already installed the extension, it should update automatically within the next couple of hours.
+
+Standalone users (e.g. React Native or Safari) can install the new version [from NPM](https://www.npmjs.com/package/react-devtools):
```
Comment (gaearon) [inline content/blog/2019-08-15-new-react-devtools.md]: Maybe "If you use the standalone shell (e.g. in React Native or Safari), you can [...]"

"Standalone users" is odd

File: content/blog/2019-08-15-new-react-devtools.md [hunk 11]
```diff
@@ -0,0 +1,73 @@
...
+
+```shell
+npm install -g react-devtools@^4
+```
+
+## Where did all of the DOM elements go?
+
+The new DevTools provides a way to filter components from the tree to make it easier to navigate large applications.
```
Comment (gaearon) [inline content/blog/2019-08-15-new-react-devtools.md]: This sounds like we're biased towards larger apps. Maybe:

>The new DevTools provides a way to filter components from the tree to make it easier to navigate deeply nested hierarchies.

File: content/blog/2019-08-15-new-react-devtools.md [hunk 12]
```diff
@@ -0,0 +1,73 @@
...
+# Follow the on-screen instructions to complete installation
+```
+
+## Thank you!
+
+We'd like to thank everyone who tested the early release of DevTools version 4.
+Your feedback helped improve this initial release significantly.
+
+We still have many exciting features planned and feedback is always welcome!
+Please feel free to open a [GitHub issue](https://github.com/bvaughn/react-devtools-experimental/issues/new) or tag [@brian\_d\_vaughn on Twitter](https://twitter.com/brian_d_vaughn).
```
Comment (gaearon) [inline content/blog/2019-08-15-new-react-devtools.md]: Oof, you're brave. Can put just `reactjs` account there.

Re: issues, probably want to update the link?

File: content/blog/2019-08-15-new-react-devtools.md [hunk 13]
```diff
@@ -0,0 +1,73 @@
...
+title: "Introducing the New React DevTools"
+author: [bvaughn]
+---
+We are excited to announce a new release of the React Developer Tools, available today in Chrome, Firefox, and (Chromium) Edge!
+
+## What's changed?
+
+A lot has changed in version 4!
+At a high level, this new version should offer significant performance gains and an improved navigation experience.
+
```
Comment (gaearon) [inline content/blog/2019-08-15-new-react-devtools.md]: Just one screenshot or a gif here, pretty please? I need to see what I'm gonna be using.

File: content/blog/2019-08-15-new-react-devtools.md (hunk 9 above)
Comment (bvaughn) [reply content/blog/2019-08-15-new-react-devtools.md]: Sounds good

File: content/blog/2019-08-15-new-react-devtools.md (hunk 11 above)
Comment (bvaughn) [reply content/blog/2019-08-15-new-react-devtools.md]: I thought "larger apps" might be more meaningful to beginners or non-native speakers than "deeply nested hierarchies". I don't feel strongly about this though.

File: content/blog/2019-08-15-new-react-devtools.md (hunk 12 above)
Comment (bvaughn) [reply content/blog/2019-08-15-new-react-devtools.md]: 😆 Sure

File: content/blog/2019-08-15-new-react-devtools.md (hunk 13 above)
Comment (bvaughn) [reply content/blog/2019-08-15-new-react-devtools.md]: Screen Shot 2019-08-15 at 10 44 41 AM
